### PR TITLE
Add jedi-cmake to bundle.

### LINKED ISSUE #579: Build soca with static netCDF library.
## Description
This is not necessarily a bug or feature in SOCA.  
SOCA bundle will not build completely when the netcdf libraries it finds are static.
This is due to a limitation in ecbuild's FindNetCDF.cmake
It has been fixed in jedi-cmake.

## Solution
Add jedi-cmake to the bundle.

## Alternatives
Open a PR into ecbuild from jedi-cmake and bring in the FindNetCDF.cmake from there.

### PR DESCRIPTION
**Pull request by aerorahul**
## Description

Add `jedi-cmake` to the bundle

What bug does it fix, or what feature does it add?
makes soca build with static netCDF libraries.

Is a change of answers expected from this PR?
No.

Issues:
closes #579 

## Testing

How were these changes tested?
In CI
What compilers / HPCs was it tested with?
Tested on machines with static netcdf libraries.
The changes do not impact containers or stacks built with jedi-stack.
Are the changes covered by ctests? (If not, tests should be added first.) - N/A

### CODE REVIEW

**Comment by travissluka** [comment]
eh, that's not going to work. that include location is only valid if you build everything as a single bundle, which is not how our travisci tests work. also, i think the LIB_REPOS names should match the project name in the bundle, so use `jedicmake`, or change it to `jedi-cmake` in the bundle

File: bundle/CMakeLists.txt [hunk 1]
```diff
@@ -14,6 +14,8 @@ include( ecbuild_bundle )
 
 ecbuild_bundle_initialize()
 
+ecbuild_bundle( PROJECT jedicmake GIT "https://github.com/jcsda-internal/jedi-cmake.git" BRANCH develop UPDATE )
```
Comment (travissluka) [inline bundle/CMakeLists.txt]:
```suggestion
ecbuild_bundle( PROJECT jedicmake GIT "https://github.com/jcsda-internal/jedi-cmake.git" UPDATE BRANCH develop  )
```

Comment (aerorahul) [reply bundle/CMakeLists.txt]: yup noticed that and done.

**Comment by travissluka** [comment]
I guess it actually worked? I'm assuming cmake will complain if the `include()` path wasn't actually found.

**Review by travissluka — APPROVED** [review]

**Review by travissluka — APPROVED** [review]

**Pull request merged**